src/test/java/org/folio/marc4ld/mapper/Marc2BibframeMapperIT.java

```diff
@@ -1117,4 +1120,8 @@ private void validateSubjectEdge(ResourceEdge subjectEdge, List<ResourceTypeDict
     assertThat(edgeIterator.hasNext()).isFalse();
   }
 
+  private void validateId(Resource resource) {
+    var expectedId = hashService.hash(resource);
+    assertThat(resource.getId()).isEqualTo(expectedId);
```

Comment:
Here we are asserting that the application invoked ` hashService.hash(resource)` after setting all the properties of the resource.